src/counts.jl

```diff
@@ -351,6 +385,9 @@ of occurrences.
                      RAM and is safe for any data type.
 """
 countmap(x::AbstractArray{T}; alg = :auto) where {T} = addcounts!(Dict{T,Int}(), x; alg = alg)
+# specialist function for Bool
+countmap(x::AbstractArray{T}) where T<:Union{Bool, UInt8, UInt16, Int8, Int16} = addcounts!(Dict{T,Int}(), x) 
```

Comment:
You can remove this function after adding support for the `alg` keyword argument to `addcounts!` above.